app/hooks/use-display-currency.ts

```diff
@@ -173,6 +179,56 @@ export const useDisplayCurrency = () => {
     [displayCurrencyDictionary],
   )
 
+  const getSecondaryAmountIfCurrencyIsDifferent = useCallback(
+    ({
+      primaryAmount,
+      displayAmount,
+      walletAmount,
+    }: {
+      primaryAmount: MoneyAmount<WalletOrDisplayCurrency>
+      displayAmount: DisplayAmount
+      walletAmount: WalletAmount<WalletCurrency>
+    }) => {
+      // if the display currency is the same as the wallet amount currency, we don't need to show the secondary amount (example: USD display currency with USD wallet amount)
+      if (walletAmount.currency === displayCurrency) {
+        return undefined
+      }
+
+      if (primaryAmount.currency === DisplayCurrency) {
+        return walletAmount
+      }
```

Comment:
the mix between `displayCurrency` and `DisplayCurrency` make the code hard to read/understand. 

I understand than DisplayCurrency is hardcoded to `String(DisplayCurrency)` but I'm wondering if another name could help clarify the difference? (not obvious suggestion to mind right now)

Reply:
maybe now that I get to read it multiple time, I get accustom to it.